src/Xenial.Tasty/Reporters/Reporters.NyanCat.cs

```diff
@@ -73,29 +73,29 @@ private static bool HasValidConsole
             }
         }
 
-        private static Task ReportSummary(IEnumerable<TestCase> tests)
+        private static Task ReportSummary(TestSummary summary)
         {
             Console.WriteLine();
             Console.ForegroundColor = ColorScheme.Default.DefaultColor;
-            Console.WriteLine($"\t{tests.Count()} total ({(int)Math.Round(TimeSpan.FromTicks(tests.Sum(t => t.Duration.Ticks)).TotalSeconds, 0, MidpointRounding.AwayFromZero)} s)");
+            Console.WriteLine($"\t{summary.Count()} total ({(int)Math.Round(TimeSpan.FromTicks(summary.Sum(t => t.Duration.Ticks)).TotalSeconds, 0, MidpointRounding.AwayFromZero)} s)");
```

Comment:
@biohazard999 Thoughts on potentially supporting pretty timespans for summary output? For example here you count seconds, but what if all results are a fraction. It could should XX ms or XX ns depending on the lowest value or some preset smallest unit. Something like this could help: https://humanizr.net/#humanize-timespan

if all results are in ns then it wouldn't show a fraction of

Reply:
A good example of this is BenchmarkDotNet summaries:
![image](https://user-images.githubusercontent.com/11443637/90571240-5ae0ce80-e17f-11ea-947f-5ddeff43bedd.png)

Reply:
Yeah I know humanize, the problem is, they don't support short notation yet. So we have to write it our selfs.